src/AuraApp.js

```diff
@@ -95,6 +110,58 @@ class AuraApp extends React.Component {
     });
   };
 
+  // Method used to like a business
+  likeBusinessHandler = async business => {
+    // State verifies whether you are logged in or not
+    if (this.state.isAuthenticated) {
+      // Gets token from local storage
+      const token = localStorage.getItem('auraUserToken');
+      // Reads user from the database
+      await API.get('account/read-user', {
+        token,
+      }).then(response => {
+        // To see if the user already has the business._id in his list of favorite businesses
+        const output = response.data.user.favorites.filter(
+          favorite => favorite.businessId === business._id
+        );
```

Comment:
This should be optimized to be an object. The longer this array gets, the longer the favoriting process takes.